Error: Triple:1273731
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/73B3.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/73B3.rdf> .'

Error: Triple:1437168
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/73N-G.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/73N-G.rdf> .'

Error: Triple:1438729
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/78C.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/78C.rdf> .'

Error: Triple:1450904
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/1420.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/1420.rdf> .'

Error: Triple:1455480
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/72A1-A.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/72A1-A.rdf> .'

Error: Triple:1475776
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/vending-machine/23.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/42-bar-2> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/vending-machine/23.rdf> .'

Error: Triple:1475786
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/vending-machine/23.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/42-sandwich> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/vending-machine/23.rdf> .'

Error: Triple:1475831
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/vending-machine/23.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/42-cafe> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_4_en-GB"@EN-GB <http://data.southampton.ac.uk/vending-machine/23.rdf> .'

Error: Triple:1475876
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/vending-machine/23.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/42-cube-food> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/vending-machine/23.rdf> .'

Error: Triple:1475887
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/vending-machine/23.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/42-piazza> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/vending-machine/23.rdf> .'

Error: Triple:1475928
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/vending-machine/23.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/42-cube-bar> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_5_en-GB"@EN-GB <http://data.southampton.ac.uk/vending-machine/23.rdf> .'

Error: Triple:1475944
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/vending-machine/23.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/42-bridge-bar> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/vending-machine/23.rdf> .'

Error: Triple:1475982
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/vending-machine/23.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/42-bridge-food> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/vending-machine/23.rdf> .'

Error: Triple:1482211
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/1730-20.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/1730-20.rdf> .'

Error: Triple:1493270
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/1612.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/1612.rdf> .'

Error: Triple:1498881
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/73N-B.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/73N-B.rdf> .'

Error: Triple:1505727
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/94B.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/94B.rdf> .'

Error: Triple:1506387
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/facility/F0004.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/facility/F0004.rdf> .'

Error: Triple:1548660
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/bicycle-parking-57.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/163-university-road> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/bicycle-parking-57.rdf> .'

Error: Triple:1565644
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/1730-42.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/1730-42.rdf> .'

Error: Triple:1567197
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/81B.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/81B.rdf> .'

Error: Triple:1609974
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/1503.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/1503.rdf> .'

Error: Triple:1639040
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/1603.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/1603.rdf> .'

Error: Triple:1645423
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN-GB <http://data.southampton.ac.uk/point-of-service/bicycle-parking-15.rdf> .):
u'<http://id.southampton.ac.uk/point-of-service/163-university-road> <http://purl.org/openorg/ukfhrsRatingKey> "fhrs_3_en-GB"@EN-GB <http://data.southampton.ac.uk/point-of-service/bicycle-parking-15.rdf> .'

Error: Triple:1672232
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/facility/F0026.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/facility/F0026.rdf> .'

